CHANGELOG.md

```diff
@@ -10,6 +10,13 @@ The following changes are pending, and will be applied on the next major release
 
 ## Unreleased
 
+### Bugfixes
+
+- `denyAccessRequest` didn't normalize the returned denied Access Grant, resulting in it having a
+  JSON-LD frame different from the value returned by `approveAccessRequest`. The value is now normalized,
+  and both functions return a similarly shaped object. This also fixes the return type of `denyAccessRequest`,
+  which was looser than it should have been.
```

Comment:
```suggestion
  and both functions return a similarly shaped object. This also fixes the return type of `denyAccessRequest`,
  which now returns the more strict `AccessGrant` type rather than the `VerifiableCredential` type.
```